src/platform_impl/ios/view.rs

```diff
@@ -123,17 +123,26 @@ unsafe fn get_view_class(root_view_class: &'static Class) -> &'static Class {
 
                 let window: id = msg_send![object, window];
                 assert!(!window.is_null());
-                let bounds: CGRect = msg_send![window, bounds];
+                let window_bounds: CGRect = msg_send![window, bounds];
                 let screen: id = msg_send![window, screen];
                 let screen_space: id = msg_send![screen, coordinateSpace];
                 let screen_frame: CGRect =
-                    msg_send![object, convertRect:bounds toCoordinateSpace:screen_space];
+                    msg_send![object, convertRect:window_bounds.clone() toCoordinateSpace:screen_space];
```

Comment:
Since we already have the window bounds in scope, I decided to just clone here to avoid the borrow and re-use it down below. I'm new to using rust so still figuring my way around the borrow checker.

Reply:
`CGRect` should probably just derive `Copy`, since it's just 4 floats.

Reply:
Thanks updated. I'm new to rust so when I saw `Clone` on `CGRect` I thought that meant `Copy` and was wondering why I still needed to clone to avoid the borrow. Now I know the difference between `Copy` vs `Clone`. 👍

Reply:
Just realised I've said I'm new to rust twice in the same comment thread. Did I mention that I'm n...